src/asim/configs/resident/trip_mode_choice.csv

```diff
@@ -292,102 +292,102 @@ util_KNR_MIX - Destination Employment Density,KNR_MIX - Destination Employment D
-util_TNCIN_PRM_transfer_wait_time,TNCIN_PRM - transfer wait time,"@(odt_skims['TNCIN_PRM_XFERWAIT']) * df.time_factor * ~df.outbound * np.where(odt_skims['TNCIN_PRM_BRTIVTT'] > 0,WAIT_TIME_DISC,1)",,,,,,,,,,,,,,,,coef_xwait,,,,,,,
-util_TNCIN_PRM_number_of_transfers,TNCIN_PRM - number of transfers,"@(-23+23*np.exp(0.414*np.clip(odt_skims['TNCIN_PRM_XFERS'] + df.mtnev_acc_xfer_in + df.mtnev_egr_xfer_in, a_min=None,a_max=4))) * df.time_factor * ~df.outbound",,,,,,,,,,,,,,,,coef_xferdrive,,,,,,,
-util_TNCIN_PRM_TNCIN_time,TNCIN_PRM - TNC time,"@np.where(df.nev_prm_egress_available_in, df.nev_prm_egress_time_in, np.where(df.microtransit_prm_egress_available_in, df.microtransit_prm_egress_time_in, odt_skims['TNCIN_PRM_EGR'])) * df.time_factor * ~df.outbound",,,,,,,,,,,,,,,,coef_acctime,,,,,,,
-util_TNCIN_PRM_TNCIN_wait_time,TNCIN_PRM - TNC wait time,"@np.where(df.nev_prm_egress_available_in, nevWaitTime, np.where(df.microtransit_prm_egress_available_in, microtransitWaitTime, df.destSingleTNCWaitTime)) * df.time_factor * ~df.outbound",,,,,,,,,,,,,,,,coef_wait,,,,,,,
-util_TNCIN_PRM_TNCIN_fare,TNCIN_PRM - TNC fare,"@(~df.nev_prm_egress_available_in) * (~df.microtransit_prm_egress_available_in) * (~df.outbound) * ((np.maximum(TNC_single_baseFare + (odt_skims['TNCIN_PRM_EGR']/60 * driveSpeed) * TNC_single_costPerMile + odt_skims['TNCIN_PRM_EGR'] * TNC_single_costPerMinute, TNC_single_costMinimum) * 100) / (np.maximum(df.income,1000)**df.income_exponent))",,,,,,,,,,,,,,,,coef_income,,,,,,,
-util_TNCIN_PRM_Walk_access_time_(at_attraction_end),TNCIN_PRM - Walk access time (at attraction end),"@np.where(df.nev_prm_access_available_in, df.nev_prm_access_time_in, np.where(df.microtransit_prm_access_available_in, df.microtransit_prm_access_time_in, df.origin_prm_time)) * df.time_factor * ~df.outbound",,,,,,,,,,,,,,,,coef_acctime,,,,,,,
-util_TNCIN_PRM_wait_access_time_(at_attraction_end),TNCIN_PRM - Access mt/nev wait time (at attraction end),"@np.where(df.nev_prm_access_available_in, nevWaitTime, np.where(df.microtransit_prm_access_available_in, microtransitWaitTime, 0)) * df.time_factor * ~df.outbound",,,,,,,,,,,,,,,,coef_wait,,,,,,,
-util_TNCIN_PRM_Walk_other_time,TNCIN_PRM - Walk other time,@(odt_skims['TNCOUT_PRM_XFERWALK']) * df.time_factor * ~df.outbound,,,,,,,,,,,,,,,,coef_xwalk,,,,,,,
-util_TNCIN_PRM_Fare,TNCIN_PRM - Fare,"@df.transitSubsidyPassDiscount*(odt_skims['TNCIN_PRM_FARE'])*100*df.number_of_participants/(np.maximum(df.income,1000)**df.income_exponent) * ~df.outbound",,,,,,,,,,,,,,,,coef_income,,,,,,,
-util_TNCIN_PRM_ASC,TNCIN_PRM - Alternative-specific constant,@df.tnc_prm_inb_asc * df.time_factor * ~df.outbound,,,,,,,,,,,,,,,,coef_ivt,,,,,,,
+util_KNRIN_PRM_In_vehicle_time,KNRIN_PRM - In-vehicle time,@(odt_skims['KNRIN_PRM_TOTALIVTT']) * df.time_factor * ~df.outbound,,,,,,,,,,,,,,,,coef_ivt,,,,,,,
```

Comment:
Labels and descriptions were incorrectly changed here, presumably through find-and-replace. No impact on functionality but could cause confusion/

Reply:
fixed.